examples/equation/equation.html

```diff
@@ -304,12 +305,13 @@
     })
   }
 
-  componentWillMount() {
+  componentDidMount() {
```

Comment:
We might be able to do this before mount if we create the Draft Editor first. That way we can call `getEditorKey()`. That way we don't end up with a double render.